sellercenter-translation/src/main/java/com/gfg/sellercenter/translation/reader/HttpReader.java

```diff
@@ -0,0 +1,45 @@
+package com.gfg.sellercenter.translation.reader;
+
+import lombok.AllArgsConstructor;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+
+@AllArgsConstructor
+public class HttpReader {
+
+    private final String fqdnUrl;
+
+    private final String query;
+
+    public JSONObject getTranslation(String language, String key) throws IOException {
+        return new JSONObject(this.loadResponse(language, key));
+    }
+
+    private String loadResponse(String language, String key) throws IOException
+    {
+        URL url = new URL(this.fqdnUrl + this.query + "?language=" + language + "&key=" + key);
```

Comment:
It is strongly recommended to not build URL's manually and use so called URI Builders:

```
UriComponentsBuilder.newInstance()
  .scheme(scheme)
  .host(host)
  .path(path)
  .build()
  .toUri()
  .toURL();
```
(This is for spring)

this is safer, and better readablle in general